ouroboros-consensus/test/consensus-test/Test/Consensus/HardFork/Infra.hs

```diff
@@ -111,6 +111,7 @@ genEraParams = do
     eraEpochSize  <- EpochSize         <$> choose (1, 10)
     eraSlotLength <- slotLengthFromSec <$> choose (1, 5)
     eraSafeZone   <- genSafeZone
+    eraGenesisWin <- GenesisWindow     <$> choose (1, 10)
```

Comment:
Doesn't need to be the same as the safe zone? Or the same as some stability window?

Reply:
Right, I think we can just use the safe zone here.
As long as these tests don't care about Genesis I think there's no reason to be flexible here.